### PR TITLE
Group SteamID viewer

### PR DESCRIPTION
**Pull request by Revadike**
image

Resolves https://github.com/tfedor/AugmentedSteam/issues/405#issuecomment-546754901
Closes https://github.com/tfedor/AugmentedSteam/pull/778/

### CODE REVIEW

File: js/content/common.js [hunk 1]
```diff
@@ -1010,25 +1010,57 @@ let SteamId = (function(){
...
+        function copySteamId(e) {
+            let elem = e.target.closest(".es_copy");
+            if (!elem) { return; }
+    
+            Clipboard.set(elem.innerText);
+    
+            let header = document.querySelector("#es_copy_header");
+            let headerText = header.innerText;
+            header.innerText = `${Localization.str.copied}`;
+            setTimeout(() => header.innerText = headerText, 1000);
```
Comment (MxtOUT) [inline js/content/common.js]: I still think that's not necessary

File: js/core.js [hunk 2]
```diff
@@ -292,6 +292,7 @@ class UpdateHandler {
 
         if (oldVersion.isSameOrBefore("1.4.1")) {
             SyncedStorage.set("profile_steamid", SyncedStorage.get("profile_permalink"));
+            SyncedStorage.set("group_steamid", SyncedStorage.get("profile_permalink"));
```
Comment (MxtOUT) [inline js/core.js]: That setting is not related to the old one

File: js/core.js [hunk 3]
```diff
@@ -681,6 +682,7 @@ SyncedStorage.defaults = {
     'profile_backpacktf': true,
     'profile_astatsnl': true,
     'profile_steamid': true,
+    'group_steamid': true,
```
Comment (MxtOUT) [inline js/core.js]:
```suggestion
    'group_steamid': false,
```
Even more niche use case than the profile ID viewer imo

File: js/content/common.js (hunk 1 above)
Comment (Revadike) [reply js/content/common.js]: I think otherwise. How else do you know the difference between a successful copy or a missclick.

File: js/core.js (hunk 2 above)
Comment (Revadike) [reply js/core.js]: if u say so

File: js/content/common.js (hunk 1 above)
Comment (MxtOUT) [reply js/content/common.js]: It's not like you have to do a pixel perfect click and the additional action of closing the window (that is useless now most likely) gets annoying

**Comment by tfedor** [comment]
Implemented in a different way
